web/src/app/services/data-store/data-store.service.ts

```diff
@@ -455,4 +451,12 @@ export class DataStoreService {
     });
     return tasksMap;
   }
+
+  private filterConversionErrorAndLogError<T>(entityOrError: T | Error) {
```

Comment:
Nit: `filterAndLogError` (the additional "Error" is clear from context, plus there's nothing specific to "conversion" here)

Reply:
Done.